src/main/java/longtimenosee/logic/commands/PolicyAssignCommand.java

```diff
@@ -27,8 +27,8 @@ public class PolicyAssignCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": assigns a policy to a client. "
             + "Parameters: "
-            + "(Index of client) "
-            + "(Index of policy) "
```

Comment:
![image](https://user-images.githubusercontent.com/97422591/199453629-a9e46075-9957-4b47-a8c4-5b109f5c2951.png)

Newline should be added for the message usage

Reply:
Parameters and example seems to be mixed up in here. (There's no example, but parameter seems to be the example)

![image](https://user-images.githubusercontent.com/97422591/199454100-d90ae460-a94b-4dcb-9828-0a68ab8fae8e.png)